### PR TITLE
【Hackathon 4th No.15】Modify GaussianNLLLoss rfc file.

### PR DESCRIPTION
**Pull request by Atlantisming**
repair Target to Label.

代码及英文文档：https://github.com/PaddlePaddle/Paddle/pull/50843
中文文档：https://github.com/PaddlePaddle/docs/pull/5623
之前的rfc文档：https://github.com/PaddlePaddle/community/pull/446

### CODE REVIEW

File: rfcs/APIs/20230222_api_design_for_gaussiannllloss.md [hunk 1]
```diff
@@ -120,19 +120,19 @@ def gaussian_nll_loss(
...
+    label,
     var,
     full=False,
     eps=1e-6,
     reduction: str="mean",
     name:str=None,
 ) -> Tensor:`
  - Input(Tensor): 期望服从高斯分布的输入，形状为`(N, *)` 或 `(*)` 其中 `*`表示任何数量的额外维度。
- - Target(Tensor):为高斯分布的采样值，形状为`(N, *)` 或 `(*)`，与输入的形状相同， 
+ - Label(Tensor):为高斯分布的采样值，形状为`(N, *)` 或 `(*)`，与输入的形状相同，
```
Comment (GGBond8488) [inline rfcs/APIs/20230222_api_design_for_gaussiannllloss.md:131]: 参数名称全部小写， 参考https://www.paddlepaddle.org.cn/documentation/docs/zh/dev_guides/api_contributing_guides/api_design_guidelines_standard_cn.html 此处API命名以及设计规范

File: rfcs/APIs/20230222_api_design_for_gaussiannllloss.md [hunk 2]
```diff
@@ -120,19 +120,19 @@ def gaussian_nll_loss(
...
     full=False,
     eps=1e-6,
     reduction: str="mean",
     name:str=None,
 ) -> Tensor:`
  - Input(Tensor): 期望服从高斯分布的输入，形状为`(N, *)` 或 `(*)` 其中 `*`表示任何数量的额外维度。
- - Target(Tensor):为高斯分布的采样值，形状为`(N, *)` 或 `(*)`，与输入的形状相同， 
+ - Label(Tensor):为高斯分布的采样值，形状为`(N, *)` 或 `(*)`，与输入的形状相同， 
 或与输入的形状相同但有一个维度等于1（允许广播）。
  - Var(Tensor): 正方差张量，即数值均大于等于0的方差张量，形状为`(N, *)` 或 `(*)`，与输入的形状相同，或与输入的形状相同但有
```
Comment (GGBond8488) [inline rfcs/APIs/20230222_api_design_for_gaussiannllloss.md:133]: 不要用缩写

File: rfcs/APIs/20230222_api_design_for_gaussiannllloss.md [hunk 3]
```diff
@@ -120,19 +120,19 @@ def gaussian_nll_loss(
 共添加以下两个 API：
 
 `paddle.nn.functional.gaussian_nll_loss(input,
-    target,
+    label,
     var,
     full=False,
     eps=1e-6,
```
Comment (GGBond8488) [inline rfcs/APIs/20230222_api_design_for_gaussiannllloss.md:126]: 不要用缩写

File: rfcs/APIs/20230222_api_design_for_gaussiannllloss.md [hunk 4]
```diff
@@ -120,19 +120,19 @@ def gaussian_nll_loss(
 共添加以下两个 API：
 
 `paddle.nn.functional.gaussian_nll_loss(input,
-    target,
+    label,
     var,
```
Comment (GGBond8488) [inline rfcs/APIs/20230222_api_design_for_gaussiannllloss.md:124]: 同不要用缩写

File: rfcs/APIs/20230222_api_design_for_gaussiannllloss.md (hunk 3 above)
Comment (Atlantisming) [reply rfcs/APIs/20230222_api_design_for_gaussiannllloss.md:126]: 能麻烦再解释下吗？0 0

Comment (GGBond8488) [reply rfcs/APIs/20230222_api_design_for_gaussiannllloss.md:126]: 这里的var 以及eps

**Comment by luotao1** [comment]
缩写规范请参考：https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/dev_guides/api_contributing_guides/api_design_guidelines_standard_cn.html#leimingyufangfamingdeguifan

image

File: rfcs/APIs/20230222_api_design_for_gaussiannllloss.md (hunk 3 above)
Comment (Atlantisming) [reply rfcs/APIs/20230222_api_design_for_gaussiannllloss.md]: Done.

**Review by GGBond8488 — APPROVED** [review]
LGTM

**Review by luotao1 — APPROVED** [review]
